src/test/java/com/ericsson/ei/handlers/test/MongoDBHandlerTest.java

```diff
@@ -53,21 +58,35 @@ public void init() throws Exception {
     public void testGetDocuments() {
         ArrayList<String> documents = mongoDBHandler.getAllDocuments(dataBaseName, collectionName);
         assertTrue(documents.size() > 0);
+        dropCollection();
```

Comment:
No since it is not executed after each test, and will then fail on test that use mocks. All tests here "should" use mocks and the other ones should be integration tests, but that is out of this scope.

Reply:
Use then @After,,,

Reply:
That will still fail the test that cannot execute the code.